### PR TITLE
Release notes 0.6.1

### PR DESCRIPTION
**Pull request by Caleb-Hill**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+* Adjust the default log settings to squash bugs and adjust what subset of logs
+  are written out.
+
+* Move `"authorization-handler-rbac"` to default feature set.
+
+* Update the REST API tests with authorization support.
+
+### scabbard
+
+* Fix possible deadlock by shortening service lock time and adding
```
Comment (peterschwarz) [inline RELEASE_NOTES.md:28]: This should match the release notes related to this issue as seen in 0.4.3

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+
+* Add a state migrate command for migrating state between database backends.
+
+* Update stores to support write-exclusivity for database backends
+
+## libsplinter
+
+* Add validation that no SQL backed trees exist if LMDB is enabled.
+
+* Remove the `health` service.
```
Comment (peterschwarz) [inline RELEASE_NOTES.md:15]: This is not a libsplinter change

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
 # Release Notes
 
+## Changes in splinter 0.6.1
+
+### Highlights
+
+* Add a state migrate command for migrating state between database backends.
+
+* Update stores to support write-exclusivity for database backends
```
Comment (peterschwarz) [inline RELEASE_NOTES.md:9]: Some details should be included under the various sections

**Review by peterschwarz — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md (hunk 2 above)
Comment (Caleb-Hill) [reply RELEASE_NOTES.md]: Updated to match google doc

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,62 @@
...
+* Add validation that no SQL backed trees exist if LMDB is enabled.
+
+* Add `OrchestratableService` to `Orchestrator`.
+
+* Remove lock-contention in `Orchestrator` REST API calls.
+
+* REST API features have been reorganized, placing all Actix Web 1 related code
+  under the rest-api-actix-web-1 related feature.
+
+* Several public const values in crate::protocol have been moved or made
```
Comment (peterschwarz) [inline RELEASE_NOTES.md:23]: This and other module paths should use inline-code quotes

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,62 @@
...
+### libsplinter
+
+* Add validation that no SQL backed trees exist if LMDB is enabled.
+
+* Add `OrchestratableService` to `Orchestrator`.
+
+* Remove lock-contention in `Orchestrator` REST API calls.
+
+* REST API features have been reorganized, placing all Actix Web 1 related code
+  under the rest-api-actix-web-1 related feature.
```
Comment (peterschwarz) [inline RELEASE_NOTES.md:21]: Features should be double-quoted and in inline-code quotes

File: RELEASE_NOTES.md (hunk 3 above)
Comment (Caleb-Hill) [reply RELEASE_NOTES.md]: Fixed

File: RELEASE_NOTES.md (hunk 4 above)
Comment (Caleb-Hill) [reply RELEASE_NOTES.md]: Fixed

File: RELEASE_NOTES.md (hunk 5 above)
Comment (Caleb-Hill) [reply RELEASE_NOTES.md]: Fixed

**Review by peterschwarz — APPROVED** [review]

**Review by rbuysse — APPROVED** [review]

**Pull request merged**